packages/sources/intrinio/package.json

```
@@ -31,6 +31,9 @@
     "@chainlink/types": "workspace:*",
     "@types/jest": "27.0.3",
     "@types/node": "16.11.19",
+    "@types/supertest": "2.0.11",
+    "nock": "13.2.4",
+    "supertest": "6.1.6",
```

```suggestion
    "supertest": "6.2.2",
```
Renovate bot bumped this dependency version.
The tests fail because of the mismatch in checksums.